### DISCUSSION TITLE
Information about company that produces iCub's flexible pcb cards

### DISCUSSION THREAD

**Discussion by plinioMoreno** [robotology/community#73] [Q&A] [answered]
Here in Lisbon we want to request a flexible pcb like the one the iCub has on its fingertips and skin sensor (cdc004 in the iCubPlatform repository), in order to put some hall sensors on Vizzy's hands.

We want to know where did you order the boards. And the price range, because we want to pay a reasonable price.

**Comment by pattacini** [comment]
Perhaps @maggia80 has info on that.

**Comment by maggia80** [comment, marked as answer]
You can order it from [**PHOENIX S.r.l.**](http://www.phoenixpcb.it). The price depends quite a lot from the size, the number of layers and the technology used (trace dimensions, hole sizes, etc.). Therefore, you should send them the gerber to get a quotation.

**Comment by plinioMoreno** [comment]
Thanks @maggia80 for the info.
